_articles/desktop-environment.md

```diff
@@ -258,6 +258,13 @@ Or, if that doesn't stop the 2nd prompt, the screensaver can be removed with thi
 When using Gnome, the lock screens will still function normally even with the screensaver disabled.
 
 
+#### Enable Cinnamon Lock Screen
+
+If Cinnamon's desktop lock isn't working, then running this command will re-enable it:
+
+`gsettings set org.cinnamon.desktop.lockdown disable-lock-screen false`
+
+
```

Comment:
This is already in the doc, no need to add it again.  This just creates extra git confusion.